var/spack/repos/builtin/packages/libpeas/package.py

```diff
@@ -0,0 +1,101 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack import *
+import os
+
+
+class Libpeas(AutotoolsPackage):
+    """libpeas is a gobject-based plugins engine, and is targeted at
+    giving every application the chance to assume its own
+    extensibility."""
+
+    homepage = "http://developer.gnome.org/libpeas/stable"
+    url = 'https://download.gnome.org/sources/libpeas/1.22/libpeas-1.22.0.tar.xz',
```

Comment:
@goxberry I think the comma at the end of this string is valid python syntax but causes the unit tests to fail

Reply:
It's valid Python syntax, but `url` has to be a string, not a tuple.

Reply:
@scheibelp @adamjstewart Fixed in https://github.com/spack/spack/pull/12385/commits/bcc12142830f075e836cafdfa7b3983a755530c0